cmd/oras/internal/errors/errors.go

```diff
@@ -26,6 +26,15 @@ import (
 	"oras.land/oras-go/v2/registry/remote/errcode"
 )
 
+// OperationType reperesnts the type of operation.
+type OperationType string
+
+const (
+	// ParseArtifactReference is the operation type parsing artifact
+	// reference.
+	ParseArtifactReference = OperationType("")
```

Comment:
Set this to `ParseArtifactReference` maybe?

Reply:
Sorry I don't get it.

Reply:
Do you mean change the string value to `"ParseArtifactReference "`

Reply:
Yes or maybe use iota instead?